### DISCUSSION TITLE
RockyLinux 8.10でのインストールについて

### DISCUSSION THREAD

**Discussion by dendenvinum** [SeciossOpenSource/LISM#5] [Q&A]
RockyLinux 8.10の環境で、README.mdの通りにインストールしていますが、install.sh installを行ったところで、

***********************************************
引き続き初期設定ツールを起動します。よろしいですか？ [yes]
yes
Can't locate Cache/Memcached/libmemcached.pm in @INC (you may need to install the Cache::Memcached::libmemcached module) (@INC contains: /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5) at ./setup.pl line 30.
***********************************************

となってしまいます。setup.plを実施しても同じメッセージとなります。
perl-Cache-Memcached.noarchのパッケージをインストールしても、対象のモジュールは存在しませんでした。

前提パッケージとして、Cache::Memcached::libmemcachedが必要なことは分かりますが、対応方法を教えていただくことは可能でしょうか？

**Comment by dendenvinum** [comment]
自己解決しました。

install.sh installを行う前に

perl-String-Random.noarch
perl-XML-Twig.noarch
perl-Cache-Memcached.noarch
のパッケージと
Cache::Memcached::libmemcachedをCPAN経由でインストールが必要でした。

インストールをしていってますが、途中で

# cp init.389ds.xxxx.ldif modify.ldif

をするにあたり、対象が

init.389ds.sime.ldif
init.389ds.siam.ldif
init.389ds.same.ldif

とみっつ存在しました。これは何が必要であれば、どれをコピーして、BaseDNを変更して、LDAP登録するのかはどう判断すればいいのでしょうか？

**Comment by secioss-git** [comment]
お問い合わせありがとうございます。

セシオスの関口と申します。

弊社製品で利用している初期データを流用しておりますので無駄もありますが
init.389ds.sime.ldif をご利用ください。

もちろん、この初期データを使わずにご自身でベースDNなどを登録いただいても構いません。

以上よろしくお願い申し上げます。

2024年8月22日(木) 9:41 'dendenvinum' via 情報 ***@***.***>:

> 自己解決しました。
>
> install.sh installを行う前に
>
> perl-String-Random.noarch
> perl-XML-Twig.noarch
> perl-Cache-Memcached.noarch
> のパッケージと
> Cache::Memcached::libmemcachedをCPAN経由でインストールが必要でした。
>
> インストールをしていってますが、途中で
> cp init.389ds.xxxx.ldif modify.ldif
>
> をするにあたり、対象が
>
> init.389ds.sime.ldif
> init.389ds.siam.ldif
> init.389ds.same.ldif
>
> とみっつ存在しました。これは何が必要であれば、どれをコピーして、BaseDNを変更して、LDAP登録するのかはどう判断すればいいのでしょうか？
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SeciossOpenSource/LISM/discussions/5#discussioncomment-10413650>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACT2E3OO7OWCMMMTWZIF75DZSUXS3AVCNFSM6AAAAABMZBXR5CVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANBRGM3DKMA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 
========================================
関口 克己
E-Mail: ***@***.***

株式会社セシオス
〒171-0022
東京都豊島区南池袋3-14-11
中町ビル6F

TEL   03-6877-5217
FAX   03-6674-1267
URL: https://www.secioss.co.jp
========================================
